### PR TITLE
CLI HELP: import 

### PR DESCRIPTION
**Pull request by makalaaneesh**

### CODE REVIEW

File: yb-voyager/cmd/importData.go [hunk 1]
```diff
@@ -64,8 +64,8 @@ var TableNameToSchema map[string]map[string]map[string]string
 
 var importDataCmd = &cobra.Command{
 	Use:   "data",
-	Short: "This command imports data into YugabyteDB database",
-	Long:  `This command will import the data exported from the source database into YugabyteDB database.`,
+	Short: "Import data into target YugabyteDB database",
+	Long:  `Import the data exported from the source database into the target IYugabyteDB database.`,
```
Comment (priyanshi-yb) [inline yb-voyager/cmd/importData.go:68]: YugabyteDB?

File: yb-voyager/cmd/import.go [hunk 2]
```diff
@@ -212,10 +213,9 @@ func registerImportDataFlags(cmd *cobra.Command) {
...
 		"number of parallel copy command jobs to target database. "+defaultParallelismMsg)
 	BoolVar(cmd.Flags(), &tconf.EnableUpsert, "enable-upsert", true,
-		"true - to enable UPSERT mode on target tables\n"+
-			"false - to disable UPSERT mode on target tables")
+		"Enable UPSERT mode on target tables")
 	BoolVar(cmd.Flags(), &tconf.UsePublicIP, "use-public-ip", false,
-		"true - to use the public IPs of the nodes to distribute --parallel-jobs uniformly for data import (default false)\n"+
+		"Use the public IPs of the nodes to distribute --parallel-jobs uniformly for data import (default false)\n"+
 			"Note: you might need to configure database to have public_ip available by setting server-broadcast-addresses.\n"+
 			"Refer: https://docs.yugabyte.com/latest/reference/configuration/yb-tserver/#server-broadcast-addresses")
```
Comment (priyanshi-yb) [inline yb-voyager/cmd/import.go:220]: Can you pls update this docs link to - 
Refer: https://docs.yugabyte.com/preview/reference/configuration/yb-tserver/#server-broadcast-addresses

File: yb-voyager/cmd/importDataFileCommand.go [hunk 3]
```diff
@@ -348,10 +348,10 @@ func init() {
 	registerImportDataFlags(importDataFileCmd)
```
Comment (priyanshi-yb) [inline yb-voyager/cmd/importDataFileCommand.go:349]: can we add somewhere in the description of this command to refer to the [Examples](https://docs.yugabyte.com/preview/yugabyte-voyager/reference/bulk-data-load/import-data-file/#examples) section for detailed Examples.

**Review by priyanshi-yb — APPROVED** [review]
LGTM.

File: yb-voyager/cmd/import.go [hunk 4]
```diff
@@ -128,7 +129,7 @@ func registerTargetDBConnFlags(cmd *cobra.Command) {
 		"provide target SSL Certificate Path")
 
 	cmd.Flags().StringVar(&tconf.SSLMode, "target-ssl-mode", "prefer",
-		"specify the target SSL mode out of - disable, allow, prefer, require, verify-ca, verify-full")
+		"specify the target SSL mode: (disable, allow, prefer, require, verify-ca, verify-full)")
```
Comment (amit-yb) [inline yb-voyager/cmd/import.go:132]: Should we also include a link to any page (YB/PG) that describes these options in detail?

File: yb-voyager/cmd/import.go [hunk 5]
```diff
@@ -137,7 +138,7 @@ func registerTargetDBConnFlags(cmd *cobra.Command) {
 		"target SSL Root Certificate Path")
 
 	cmd.Flags().StringVar(&tconf.SSLCRL, "target-ssl-crl", "",
-		"target SSL Root Certificate Revocation List (CRL)")
+		"target SSL Root Certificate Revocation List (CRL) Path")
```
Comment (amit-yb) [inline yb-voyager/cmd/import.go:141]: "Path of the file containing target DB SSL Root ..."

File: yb-voyager/cmd/import.go [hunk 6]
```diff
@@ -224,33 +224,31 @@ func registerImportDataFlags(cmd *cobra.Command) {
...
-		"true - to disable transactional writes in tables for faster data ingestion (default false)\n"+
+		"Disable transactional writes in tables for faster data ingestion (default false)\n"+
 			"(Note: this is a interim flag until the issues related to 'yb_disable_transactional_writes' session variable are fixed. Refer: https://github.com/yugabyte/yugabyte-db/issues/12464)")
 	// Hidden for beta2.0 release (and onwards until further notice).
 	cmd.Flags().MarkHidden("disable-transactional-writes")
 
 	BoolVar(cmd.Flags(), &truncateSplits, "truncate-splits", true,
-		"true - to truncate splits after importing\n"+
-			"false - to not truncate splits after importing (required for debugging)")
+		"Truncate splits after importing")
```
Comment (amit-yb) [inline yb-voyager/cmd/import.go:233]: Include default in the message.

File: yb-voyager/cmd/import.go [hunk 7]
```diff
@@ -224,33 +224,31 @@ func registerImportDataFlags(cmd *cobra.Command) {
...
 	cmd.Flags().MarkHidden("truncate-splits")
 }
 
 func registerImportSchemaFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&tconf.ImportObjects, "object-list", "",
-		"list of schema object types to include while importing schema")
+		"comma separated list of schema object types to include while importing schema")
```
Comment (amit-yb) [inline yb-voyager/cmd/import.go:239]: Also mention whether the names are case-sensitive?

**Review by amit-yb — APPROVED** [review]
LGTM

File: yb-voyager/cmd/import.go (hunk 7 above)
Comment (makalaaneesh) [reply yb-voyager/cmd/import.go:243]: Asked @ShivanshGahlot  to take care of this as part of https://github.com/yugabyte/yb-voyager/pull/1147/files

File: yb-voyager/cmd/import.go (hunk 2 above)
Comment (priyanshi-yb) [reply yb-voyager/cmd/import.go:213]: oh okay, then it's fine. I didn't check that, I thought it was a broken link as Aishwarya shared this change when she was looking at the help msg.

**Pull request merged**